llvm/test/MC/Disassembler/ARM/sub-sp-imm-thumb2.txt

```diff
@@ -6,32 +6,32 @@
 
 # CHECK:      subw sp, sp, #1148
 # CHECK-SAME: <MCInst #{{[0-9]+}} t2SUBspImm12
-# CHECK-NEXT: <MCOperand Reg:[[SP:[0-9]+]]>
-# CHECK-NEXT: <MCOperand Reg:[[SP]]>
+# CHECK-NEXT: <MCOperand Reg:SP>
+# CHECK-NEXT: <MCOperand Reg:SP>
 # CHECK-NEXT: <MCOperand Imm:1148>
 # CHECK-NEXT: <MCOperand Imm:14>
-# CHECK-NEXT: <MCOperand Reg:0>>
+# CHECK-NEXT: <MCOperand Reg:>>
 
 0xad 0xf2 0x7c 0x4d
 
 # CHECK:      sub.w   sp, sp, #1024
 # CHECK-SAME: <MCInst #{{[0-9]+}} t2SUBspImm
-# CHECK-NEXT: <MCOperand Reg:[[SP]]>
-# CHECK-NEXT: <MCOperand Reg:[[SP]]>
+# CHECK-NEXT: <MCOperand Reg:SP>
+# CHECK-NEXT: <MCOperand Reg:SP>
 # CHECK-NEXT: <MCOperand Imm:1024>
 # CHECK-NEXT: <MCOperand Imm:14>
-# CHECK-NEXT: <MCOperand Reg:0>
-# CHECK-NEXT: <MCOperand Reg:0>>
+# CHECK-NEXT: <MCOperand Reg:>
```

Comment:
Should I add extra logic to print `NoRegister` here? Or is the empty value fine?